pkg/vault/contracts/BaseHooks.sol

```diff
@@ -84,7 +85,7 @@ abstract contract BaseHooks is IHooks, VaultGuard {
         uint256[] memory,
         uint256[] memory,
         bytes memory
-    ) external virtual onlyVault returns (bool) {
+    ) public virtual onlyVault returns (bool) {
```

Comment:
```suggestion
    ) public virtual returns (bool) {
```
Should we remove onlyVault from BaseHooks, since it's an abstract contract?

Reply:
Yes; must have missed that one.